src/main/java/emissary/config/ServiceConfigGuide.java

```diff
@@ -845,6 +846,20 @@ public Map<String, Set<String>> findStringMatchMultiMap(@Nullable final String p
         return theHash;
     }
 
+    /**
+     * Return the first string entry matching the key parameter
+     *
+     * @param theParameter key to match
+     * @return the first matching value
+     * @throws IllegalArgumentException if no non-blank value is found
+     */
+    @Override
+    public String findRequiredStringEntry(final String theParameter) throws IllegalArgumentException {
```

Comment:
Please add unit test coverage that includes one or more valid entries, a zero-length entry (`FOO = ""`), and a missing entry

Reply:
Got it.